README.md

```diff
@@ -75,7 +79,7 @@ Now you can start Firefox from your code:
 $driver = RemoteWebDriver::create($host, DesiredCapabilities::firefox());
 ```
 
-##### You can also customize the desired capabilities
+### Customizing Desired Capabilities
```

Comment:
This could be "Customize Desired Capabilities" to make it aligned with wording of the previous two headers on the same level.